tests/src/whisk/core/cli/test/WskCoreBasicTests.scala

```diff
@@ -0,0 +1,199 @@
+                (pkg, _) =>
+                    pkg.create(samplePackage, shared = Some(true))
+            }
+            assetHelper.withCleaner(wsk.action, sampleAction) {
+                val file = Some(TestUtils.getTestActionFilename("empty.js"))
+                (action, _) => action.create(sampleAction, file, kind = Some("nodejs"), shared = Some(true))
+            }
+
+            val fullyQualifiedActionName = s"/$otherNamespace/notallowed"
+            assetHelper.withCleaner(wsk.action, fullyQualifiedActionName, confirmDelete = false) {
+                (action, _) => action.create(fullyQualifiedActionName, None, update = true,
+                                             shared = Some(true), expectedExitCode = FORBIDDEN)(originWskProps)
+            }
+    }
+
+    behavior of "Wsk Package CLI"
+
+    it should "list shared packages" in withAssetCleaner(otherWskProps) {
+        (wp, assetHelper) =>
+            assetHelper.withCleaner(wsk.pkg, samplePackage) {
+                (pkg, _) =>
+                    pkg.create(samplePackage, shared = Some(true))
+            }
+            val fullyQualifiedPackageName = s"/$otherNamespace/$samplePackage"
+            val result = wsk.pkg.list(Some(s"/$otherNamespace"))(originWskProps).stdout
+            result should include regex (fullyQualifiedPackageName + """\s+shared""")
+    }
+
+    it should "list shared package actions" in withAssetCleaner(otherWskProps) {
+        (wp, assetHelper) =>
+            assetHelper.withCleaner(wsk.pkg, samplePackage) {
+                (pkg, _) =>
+                    pkg.create(samplePackage, shared = Some(true))
+            }
+            assetHelper.withCleaner(wsk.action, sampleAction) {
+                val file = Some(TestUtils.getTestActionFilename("empty.js"))
+                (action, _) => action.create(sampleAction, file, kind = Some("nodejs"), shared = Some(true))
+            }
+            val fullyQualifiedPackageName = s"/$otherNamespace/$samplePackage"
+            val fullyQualifiedActionName = s"/$otherNamespace/$sampleAction"
+            val result = wsk.action.list(Some(fullyQualifiedPackageName))(originWskProps).stdout
+            result should include regex (fullyQualifiedActionName + """\s+shared""")
+    }
+
+    it should "create a package binding" in withAssetCleaner(otherWskProps) {
+        (wp, assetHelper) =>
+            assetHelper.withCleaner(wsk.pkg, samplePackage) {
+                (pkg, _) =>
+                    pkg.create(samplePackage, shared = Some(true))
+            }
+            val fullyQualifiedPackageName = s"/$otherNamespace/$samplePackage"
+            val name = "bindPackage"
+            val provider = fullyQualifiedPackageName
+            val annotations = Map("a" -> "A".toJson, WhiskPackage.bindingFieldName -> "xxx".toJson)
+            assetHelper.withCleaner(wsk.pkg, name) {
+                (pkg, _) =>
+                    pkg.bind(provider, name, annotations = annotations)
```

Comment:
Of course you can - as long as the package is public. That's the point of
public packages and the bind operation. What's the error you encountered?
If this is failing we have a defect.